### PR TITLE
[WIP] SWARM-1591 - Bumped OpenTracing to 0.31.0-RC1

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

- [x] Have you followed the guidelines in our [Contributing](http://wildfly-swarm.io/community/contributing/) document?
- [x] Have you created a [JIRA](https://issues.jboss.org/browse/SWARM) and used it in the commit message?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/wildfly-swarm/wildfly-swarm/pulls) for the same issue?
- [x] Have you built the project locally prior to submission with `mvn clean install`?

-----

> Have you built the project locally prior to submission with `mvn clean install`?

`mvn clean install -PskipTests` and then `mvn clean install` on the `testsuite-opentracing` module.

### CODE REVIEW

**Comment by jpkrohling** [comment]
@objectiser , could you please take a quick look at this change? Is there any other version change required for this? The tests are passing, but I wonder if any other lib might need a bump as well.

File: build-parent/pom.xml [hunk 1]
```diff
@@ -85,7 +85,7 @@
 
     <!-- OpenTracing related versions -->
     <version.opentracing.tracerresolver>0.1.0</version.opentracing.tracerresolver>
-    <version.opentracing.servlet>0.0.9</version.opentracing.servlet>
+    <version.opentracing.servlet>0.1.0-RC1</version.opentracing.servlet>
     <version.opentracing.spanmanager>0.0.5</version.opentracing.spanmanager>
```
Comment (objectiser) [inline build-parent/pom.xml]: The `spanmanager` should no longer be required as of 0.30.0 due to the introduction of `ActiveSpan`.

File: build-parent/pom.xml [hunk 2]
```diff
@@ -85,7 +85,7 @@
...
     <version.jaeger>0.20.0</version.jaeger>
```
Comment (objectiser) [inline build-parent/pom.xml]: Jaeger should be 0.22.0-RC1

**Comment by kenfinnigan** [comment]
What's the current status of this?

**Comment by objectiser** [comment]
Hopefully OT Java 0.31.0 will be released in the next week or two, so probably best to hold off and we'll update to non-RC versions at that point.

**Comment by jpkrohling** [comment]
Closing this in favor of #706

**Pull request closed without merging**